sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/Readme.md

```diff
@@ -4,28 +4,35 @@ Azure Time Series Insights provides data exploration and telemetry tools to help
 
 Azure Time Series Insights gives you a global view of your data, so you can quickly validate your IoT solution and avoid costly downtime to mission-critical devices. It can help you discover hidden trends, spot anomalies, and conduct root-cause analysis in near real time.
 
-If you are new to Azure Time Series Insights and would like to learn more about the platform, please make sure you check out the Azure Time Series Insights official documentation page.
+If you are new to Azure Time Series Insights and would like to learn more about the platform, please make sure you check out the Azure Time Series Insights official [documentation page][tsi_product_documentation].
 
-# Time Series Insights Samples
+# Time Series Insights client library for .NET
+The Time Series Insights client library for .NET provides the following functionality:
+- Retrieving and being able to make changes to the Time Series Insights environment model settings, such as changing the model name or default type ID.
+- Retrieving and being able to add, update and remove Time Series instances.
+- Retrieving and being able to make changes to the Time Series Insights environment types, such as creating, updating and deleting Time Series types.
+- Retrieving and being able to make changes to the Time Series Insights hierarchies, such as creating, updating and deleting Time Series hierarchies.
+- Executing Time series queries such as querying for raw events, series and aggregate series
```

Comment:
Is Executing right word?

One suggestion is:
Query for raw events, series and aggregate series